doc/changes/changes_1.0.2.md

```diff
@@ -0,0 +1,41 @@
+# Virtual Schema for Databricks 1.0.2, released 2025-??-??
+
+Code name: Fixed vulnerability CVE-2025-48924 in org.apache.commons:commons-lang3:jar:3.16.0:test
+
+## Summary
+
+This release fixes the following vulnerability:
+
+### CVE-2025-48924 (CWE-674) in dependency `org.apache.commons:commons-lang3:jar:3.16.0:test`
+Uncontrolled Recursion vulnerability in Apache Commons Lang.
+
+This issue affects Apache Commons Lang: Starting withÂ commons-lang:commons-langÂ 2.0 to 2.6, and, from org.apache.commons:commons-lang3 3.0 beforeÂ 3.18.0.
+
+The methods ClassUtils.getClass(...) can throwÂ StackOverflowError on very long inputs. Because an Error is usually not handled by applications and libraries, a 
```

Comment:
```suggestion
The methods ClassUtils.getClass(...) can throw a `StackOverflowError` on very long inputs. Because an Error is usually not handled by applications and libraries, a 
```